<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stat</class>
 <widget class="QDialog" name="stat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>590</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>DOCTOR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>INGENIEUR</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>PROFESSOR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_6" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget_7" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
background-color: rgb(0, 170, 0);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="title">
    <string>statistiques</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
